tests/keras/test_keras_model_export.py

```diff
@@ -204,21 +204,23 @@ def _import_module(name, **kwargs):
         path0 = os.path.join(model_path, "0")
         with pytest.raises(MlflowException):
             mlflow.keras.save_model(x, path0)
-        mlflow.keras.save_model(x, path0, keras_module=FakeKerasModule)
+        mlflow.keras.save_model(x, path0, keras_module=FakeKerasModule, save_format="h5")
```

Comment:
Specify `save_format="h5"` in tests that assume a model is saved in the H5 format.

Reply:
This implicitly ensures that we can still save a keras model in the H5 format.